options/options.go

```diff
@@ -29,18 +29,17 @@ import (
 
 // Options define common options for configuring scorecard.
 type Options struct {
-	Repo       string
-	Local      string
-	Commit     string
-	LogLevel   string
-	Format     string
-	NPM        string
-	PyPI       string
-	RubyGems   string
-	Nuget      string
-	PolicyFile string
-	// TODO(action): Add logic for writing results to file
-	ResultsFile string
```

Comment:
can we keep the option member named `ResultsFile` for backwards compatability reasons?

Reply:
Sure!